purchasing_test/unit/opportunities/test_opportunities.py

```diff
@@ -71,6 +71,35 @@ def setUp(self):
         # import our test categories
         import_nigp(current_app.config.get('PROJECT_ROOT') + '/purchasing_test/mock/nigp.csv')
 
+    def test_templates(self):
```

Comment:
Added one for opportunities_admin. Should this be in test_opportunity_admin or is the way I did it here ok?

I think the conductor test [has this covered already](https://github.com/codeforamerica/pittsburgh-purchasing-suite/blob/master/purchasing_test/unit/conductor/test_conductor.py#L466).